server/src/model/cc-license.ts

```diff
@@ -14,7 +14,7 @@ const LICENSE_ATTRIBUTES: Record<string, string> = {
   sa: 'ShareAlike'
 }
 
-const LICENSE_PATTERN = /https?:\/\/creativecommons\.org\/licenses\/([a-z-]+)\/(\d\.\d)\/?(deed\..+)?/
+const LICENSE_PATTERN = /^https?:\/\/creativecommons\.org\/licenses\/([a-z-]+)\/(\d\.\d)\/?(deed\..+)?$/
```

Comment:
Can't get the regex to work with `https://creativecommons.org/licenses/by/4.0/aaa/` on [Regex101](https://regex101.com/)

Reply:
As long as it works in the tests. Then I guess it's fine by me.

Reply:
The regex should not match `https://creativecommons.org/licenses/by/4.0/aaa/`. Prior to this change, that url would have matched because the regex did not include the beginning (`^`) and end (`$`) of the string: that was the mistake I made. Now the regex should not match unless the entire string is a valid license url.